### PR TITLE
DCAR - Update the Follow button icon and text to "Notifications on/off"

### PR DESCRIPTION
**Pull request by vlbee**
## What does this change?

This PR is to align the visuals in DCAR for contirbutor notification button on App Article Webviews  with the changes being implemented in Apps Rendering for MyGuardian. (#8920)

Full palette colours viewable in storybook. 

I pulled `splitThemeDecorateMultipleFormats` from this [approved open PR ](https://github.com/guardian/dotcom-rendering/pull/9312/files#)and have provided default multiple format values. 

## Screenshots
**BEFORE**
image

image

**AFTER**
image

[before]: https://example.com/before.png
[after]: https://example.com/after.png

### CODE REVIEW

**Review by lindseydew — COMMENTED** [review]
Looks good from what I can see. I'm not an expert in this code base, but your general approach is looking good

File: dotcom-rendering/src/components/FollowButton.tsx [hunk 1]
```diff
@@ -1,38 +1,45 @@
...
+	<div
+		css={css`
+			${isFollowing &&
+			`background-color: ${palette('--follow-icon-fill')};`}
+			height: 24px;
+			width: 24px;
+			margin: 0 0.15rem 0 0;
+			${isFollowing
+				? 'padding: 0.175rem 0 0  0.175rem;'
+				: 'padding: 0.135rem 0 0  0.125rem;'}
```
Comment (mxdvl) [inline dotcom-rendering/src/components/FollowButton.tsx:23]: Why do we have different paddings when following and not?

File: dotcom-rendering/src/palette.ts [hunk 2]
```diff
@@ -862,9 +998,17 @@ const paletteColours = {
...
-		light: followLight,
-		dark: followDark,
+	'--follow-icon-fill': {
+		light: followIconFillLight,
+		dark: followIconFillDark,
+	},
+	'--follow-icon-background': {
+		light: followIconBackgroundLight,
+		dark: followIconBackgroundDark,
+	},
```
Comment (mxdvl) [inline dotcom-rendering/src/palette.ts:1008]: I don’t fully understand the distinction between these two colours. Could you explain?

**Review by mxdvl — COMMENTED** [review]
As discussed, I think we can use the `splitTheme` decorator from #9413.

I’m a little puzzled as to why we need three distinct colours. I would imagine that the icon is simply hollowed out and we can see the background through the colourful one.

File: dotcom-rendering/src/palette.ts (hunk 2 above)
Comment (vlbee) [reply dotcom-rendering/src/palette.ts:1008]: had no idea this was a thing. great shout

Comment (mxdvl) [reply dotcom-rendering/src/palette.ts:1008]: It required quite intimate knowledge of [how SVG `fill-rule` works](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill-rule). I would love if Source offered cut-out versions of icons, but it’s definitely not a blocker for this PR!

Comment (vlbee) [reply dotcom-rendering/src/palette.ts:1008]: Going to leave as is for now until a cutout option is made available on the Source components which is the optimum solution

File: dotcom-rendering/src/components/FollowButton.tsx [hunk 3]
```diff
@@ -1,38 +1,49 @@
...
+import { remSpace, space, textSans } from '@guardian/source-foundations';
+import {
+	SvgNotificationsOff,
+	SvgNotificationsOn,
+} from '@guardian/source-react-components';
 import { palette } from '../palette';
 
 type IconProps = {
-	isFollowing: boolean;
+	isFollowing?: boolean;
```
Comment (mxdvl) [inline dotcom-rendering/src/components/FollowButton.tsx:10]: I think making this a required prop is more explicit

**Review by mxdvl — APPROVED** [review]

File: dotcom-rendering/src/components/FollowButton.tsx [hunk 4]
```diff
@@ -56,24 +61,21 @@ const containerStyles = css`
...
-	onClickHandler,
-}: Props) => {
+export const FollowButton = ({ isFollowing, onClickHandler }: Props) => {
 	return (
-		<button onClick={onClickHandler} type="button" css={buttonStyles}>
-			<span css={containerStyles}>
-				<FollowIcon isFollowing={isFollowing} />
-				<span>
-					{isFollowing ? 'Unfollow' : 'Follow'} {displayName}
+		<>
```
Comment (abeddow91) [inline dotcom-rendering/src/components/FollowButton.tsx:70]: Do we need these empty tags, it looks like the button is the parent tag here.

**Review by abeddow91 — APPROVED** [review]

**Pull request merged**